### PR TITLE
Fixed hover for clusters page, impruve speling fore go rapport card.

### LINKED ISSUE #424: Events for cluster components don't appear in the console
When mousing over (or clicking on) apps in the dashboard when a cluster is pulled up (see below example), an error appears in the JavaScript console in the inspection pane (I'm using Chrome and the current master):
```
ClusterComponent.html:77 ERROR TypeError: _co.onAppActivate is not a function
    at Object.eval [as handleEvent] (ClusterComponent.html:85)
    at handleEvent (core.js:13255)
    at callWithDebugContext (core.js:14740)
    at Object.debugHandleEvent [as handleEvent] (core.js:14327)
    at dispatchEvent (core.js:9704)
    at eval (core.js:12028)
    at SafeSubscriber.schedulerFn [as _next] (core.js:4235)
    at SafeSubscriber.__tryOrUnsub (Subscriber.js:239)
    at SafeSubscriber.next (Subscriber.js:186)
    at Subscriber._next (Subscriber.js:127)
```

The same appears regarding the listed nodes:
```
ClusterComponent.html:111 ERROR TypeError: _co.onActivate is not a function
    at Object.eval [as handleEvent] (ClusterComponent.html:119)
    at handleEvent (core.js:13255)
    at callWithDebugContext (core.js:14740)
    at Object.debugHandleEvent [as handleEvent] (core.js:14327)
    at dispatchEvent (core.js:9704)
    at eval (core.js:12028)
    at SafeSubscriber.schedulerFn [as _next] (core.js:4235)
    at SafeSubscriber.__tryOrUnsub (Subscriber.js:239)
    at SafeSubscriber.next (Subscriber.js:186)
    at Subscriber._next (Subscriber.js:127)
```

### Example:
screen shot 2018-02-14 at 3 02 56 pm

I already have a fix ready for the issue and will submit it in a later pull request. `supergiant/cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts` just needs these functions being called by `supergiant/cmd/ui/assets/src/app/clusters/cluster/cluster.component.html` to be added.

### PR DESCRIPTION
**Pull request by TheKLARKEN**
Fixes #424.

### CODE REVIEW

**Comment by TheKLARKEN** [comment]
[SG 1.0 Bug Squash: Issue 424 Events for cluster components don't appear in the console](https://trello.com/c/CEJvBsE7/336-sg-10-bug-squash-issue-424-events-for-cluster-components-dont-appear-in-the-console)

**Comment by gopherstein** [comment]
LGTM

File: test/integration/api/users_test.go [hunk 1]
```diff
@@ -168,7 +168,7 @@ func TestUsersUpdate(t *testing.T) {
 			updatedUser.Role = "admin"
 			err := sg.Users.Update(user.ID, &updatedUser)
 
-			Convey("The Update should be succesful, but the role unchanged", func() {
+			Convey("The Update should be successful, but the role unchanged", func() {
```
Comment (briansage) [inline test/integration/api/users_test.go]: 👏

File: test/integration/api/users_test.go [hunk 2]
```diff
@@ -211,7 +211,7 @@ func TestUsersUpdate(t *testing.T) {
 			updatedUser.Role = "admin"
 			err := sg.Users.Update(user.ID, &updatedUser)
 
-			Convey("The Update should be succesful, and the role changed", func() {
+			Convey("The Update should be successful, but the role unchanged", func() {
```
Comment (briansage) [inline test/integration/api/users_test.go]: 👏

File: test/integration/api/users_test.go (hunk 1 above)
Comment (TheKLARKEN) [reply test/integration/api/users_test.go]: THank you , I woekrd vety hardon this one.

**Pull request merged**